kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -53,10 +53,10 @@
       <p class="login-text no-account">{{ $tr('noAccount') }}</p>
       <div id="btn-group">
         <router-link v-if="canSignUp" class="group-btn" :to="signUp">
-          <icon-button id="signup-button" :text="$tr('createAccount')" :primary="true"/>
+          <k-button :text="$tr('createAccount')" :primary="false"/>
         </router-link>
         <a class="group-btn" href="/learn">
-          <icon-button id="guest-access-button" :text="$tr('accessAsGuest')" :primary="false"/>
+          <k-button :text="$tr('accessAsGuest')" :primary="false"/>
```

Comment:
We'll probably need some customization on these buttons because black text on white background pops too much against the dark background image.

Recommend simply inverting the buttons' text/bg color? (cc @khangmach)

Not sure the appropriate way of doing this customization while keeping the props on the button simple. we can discuss.

![image](https://user-images.githubusercontent.com/2367265/28465292-12afb874-6ddd-11e7-8a68-62f1c820350b.png)